src/mudlet.pro.user.vJvOiV

```diff
@@ -0,0 +1,590 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a QT per user/per OS local file and should NOT be include in the repository... :wastebasket:

Reply:
Feedback applied.